### DISCUSSION TITLE
Can not open properly larger .sciprj files

### DISCUSSION THREAD

**Discussion by wwwpirate** [Makhber/makhber#8] [Q&A]
I have in my computer installed Makhber version 0.9.4 released May 24, 2021 which I run on Dell laptop with windows 10.

.sciprj files created in SciDAVis version 2.8 if they are smaller than 100 kb I can open with no problem but if they are larger than that I get message 
 XML reader error: line 1 column 93954 (or any number depending on file size) missing or invalid format attribute(s) (loading fail).

I personally think that I do not have so big column numbers in my computer so I don't know where program gets those numbers from.
If code is same why is this happening in Makhber but not in SciDAVis ???

**Comment by MehdiChinoune** [comment]
SciDAVis projects will be dropped with next release 0.10.0

**Reply by MehdiChinoune** [reply]
Please, Install the latest release (0.9.7)
